src/main/java/be/seeseemelk/mockbukkit/plugin/PluginManagerMock.java

```diff
@@ -719,8 +746,7 @@ public void unsubscribeFromDefaultPerms(boolean op, Permissible permissible)
 	@Override
 	public boolean useTimings()
 	{
-		// TODO Auto-generated method stub
-		throw new UnimplementedOperationException();
+		return false;
```

Comment:
On what grounds do you return false here?
If there is a valid reason for returning a constant here then please explain so with a comment.

Reply:
Timings refer to the SpigotTimings/ Papers Timing that revolves around tick timers for reporting lag.  Its should be false in a testing environment ..because 

1. Timings are pointless in a testing environ
2.  Lag cant be used consistently anyway in a test server.

 However, this method is called internally in the JavaPluginLoader that Mockbukkit uses and causes any test that uses that call to fail.

Reply:
I agree but like I said in my review, things like that should be explained in a comment.
Good docs and comments on why something is there help make it clear for future contributors to understand why it's there.